packages/evm/test/core/controller-modules.js

```diff
@@ -604,11 +604,11 @@ contract('Controller', ([_, fundsGovernor, configGovernor, modulesGovernor, some
       const from = someone
 
       it('reverts', async () => {
-        await assertRevert(controller.linkModules([ZERO_ADDRESS], [ZERO_BYTES32], { from }), CONTROLLER_ERRORS.SENDER_NOT_GOVERNOR)
+        await assertRevert(controller.syncModuleLinks([ZERO_ADDRESS], [ZERO_BYTES32], { from }), CONTROLLER_ERRORS.SENDER_NOT_GOVERNOR)
       })
     })
 
-    context('when trying to call it directly', () => {
+    context('when trying to call linkModules directly', () => {
```

Comment:
Just wondering, should we move this test to the `Controlled` tests instead?

Reply:
I think it's fine to keep it here, I consider it a particular case of the functionality being tested